include/openenclave/edl/logging.edl

```diff
@@ -24,6 +24,8 @@ enclave
 
     untrusted
     {
+        void oe_log_is_supported_ocall();
```

Comment:
@mingweishih This is a nice idea. Consider if we can get rid of `oe_log_init_ecall` and make it an ocall that the enclave calls during initialization. That would change logging initialization from a "push" (host makes ecall) to a "pull" model (enclave makes ocall to initialize). That will get rid of ecalls that the SDK makes for logging.